packages/discord-bot/src/commands/forward.ts

```diff
@@ -49,4 +49,9 @@ export const forward: Command = {
       logger.err(err);
     }
   },
+  help: {
+    name: 'forward',
+    text: 'Command to forward praise from a giver to receivers in the discord server. You need to have an activated account on the Praise System along with FORWARDER role permissions to use this command.\n\
+  **Usage**: `/forward from: <@user1> to: <@user1 @user2 ...> reason: for something`\n',
+  },
```

Comment:
I think the `/forward` command syntax is different than noted here -- the parameters "from:" and "to" should be "giver:" and "recievers:"